[2024-06-24T11:37:08.116+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T11:37:08.141+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:37:08.153+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [queued]>
[2024-06-24T11:37:08.154+0000] {taskinstance.py:2306} INFO - Starting attempt 11 of 11
[2024-06-24T11:37:08.174+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): processing_tasks.process_data_kyiv> on 2024-06-22 00:00:00+00:00
[2024-06-24T11:37:08.182+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1618) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T11:37:08.184+0000] {standard_task_runner.py:63} INFO - Started process 1625 to run task
[2024-06-24T11:37:08.185+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'weather_dag', 'processing_tasks.process_data_kyiv', 'scheduled__2024-06-22T00:00:00+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp0wpx50j0']
[2024-06-24T11:37:08.189+0000] {standard_task_runner.py:91} INFO - Job 154: Subtask processing_tasks.process_data_kyiv
[2024-06-24T11:37:08.253+0000] {task_command.py:426} INFO - Running <TaskInstance: weather_dag.processing_tasks.process_data_kyiv scheduled__2024-06-22T00:00:00+00:00 [running]> on host afc8096e8b86
[2024-06-24T11:37:08.375+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='processing_tasks.process_data_kyiv' AIRFLOW_CTX_EXECUTION_DATE='2024-06-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='11' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-22T00:00:00+00:00'
[2024-06-24T11:37:08.378+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T11:37:08.393+0000] {python.py:237} INFO - Done. Returned value was: (1719014400, 289.55, 78, 81, 1.56)
[2024-06-24T11:37:08.394+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T11:37:08.424+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=processing_tasks.process_data_kyiv, run_id=scheduled__2024-06-22T00:00:00+00:00, execution_date=20240622T000000, start_date=20240624T113708, end_date=20240624T113708
[2024-06-24T11:37:08.480+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T11:37:08.494+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
